changed word Tokenizer

JanviRaina opened the pull request:
I made changes in the function and ran it separately and it is working fine. Let me know if this helps.

soumendrak commented on openodia/_understandData.py:
```diff
@@ -7,20 +8,40 @@
+            ]
         return token_list
 
+# import regex
+
+# def word_tokenizer(text):
+#     punct = regex.compile(r'(\p{Punctuation})')    
+#     x=punct.split(test_data)
+#     str1=""
+#     for i in x:
+#         str1+=i+' '
+#     token_list = [
+#             token
+#             for token in str1.split()
+#             if token
+#         ]
+#     return token_list
+
+# test_data = "ଭାରତୀୟ ସର୍ବୋଚ୍ଚ ନ୍ୟାୟାଳୟ, ଭାରତର ଉଚ୍ଚତମ ନ୍ୟାୟିକ ଅନୁଷ୍ଠାନ ଅଟେ ।"
+# print(word_tokenizer(test_data))
```
Commented code should not be present in the code.
If past code is needed, can be fetched from `git` commit history.
Keeping commented code reduces readability factor.

soumendrak commented on openodia/_understandData.py:
```diff
@@ -7,20 +8,40 @@
         """Split the text into words"""
-        # TODO: Do not remove the punctuation characters
+        punct = regex.compile(r'(\p{Punctuation})')    
+        x=punct.split(test_data)
```
`test_data` should be `text`

soumendrak replied:
Please give the proper variable name depending on what it is and not `x`, `str1` and `i`

soumendrak commented on openodia/_understandData.py:
```diff
@@ -7,20 +8,40 @@
+        punct = regex.compile(r'(\p{Punctuation})')    
+        x=punct.split(test_data)
+        str1=""
+        for i in x:
+            str1+=i+' '
```
This `for` loop increases additional time complexity to the codebase. 
Is there any better way to do it?

soumendrak reviewed (changes requested):
@JanviRaina 
Thanks for the changes. I really appreciate it.
Could you please make the changes requested?
Also, can you please make sure the test cases are passing? Thank you.

JanviRaina commented:
I have removed the commented portion and replaced the for loop with 'join', have tried to give some better names to the variables.
Thanks for these suggestions.
I am not sure how to test this for all the test cases though. I have only tested it using print statement for this string
("ଭାରତୀୟ ସର୍ବୋଚ୍ଚ ନ୍ୟାୟାଳୟ, ଭାରତର ଉଚ୍ଚତମ ନ୍ୟାୟିକ ଅନୁଷ୍ଠାନ ଅଟେ ।") for which it seemed to have worked fine.

soumendrak commented:
@JanviRaina run
```shell
black openodia -l 99 -t py39
```
on command prompt to get rid of the linting issue.
